### ISSUE TITLE
A11y_AzureDataStudio_Create new server group: Narrator is announcing hexadecimal value instead of name of the color

### ISSUE THREAD

**Issue by VijayaBharathyA** [microsoft/azuredatastudio#18569]
### GitHub Tags:
#A11yMAS; #A11yTCS; #A11ySev2; #DesktopApp; #BM-AzureDataStudio-Win32-Feb2022; #Azure Data Studio; #WCAG1.3.1; #Win32; #AINotInScope; #Info and Relationship; #Narrator; #NVDA;

### Environment Details:
Windows Version :Windows 11 Enterprise
Application name :Azure Data studio

### Repro Steps:
1.Turn on Narrator
2.Launch Azure Data Studio.
3.Select the Connections icon (if not already selected) from left bar.
4.Hover over the 'Servers' header and select the icon that has a tool tip of 'New Server Group' icon.
5.Enter the name for the Server group name and a description. And then a color for the group can also be selected from the provided color options. Then click 'OK'.
6.Verify whether the narrator should announced the name of the color or not .
 
### Actual Result:
Narrator is announcing hexadecimal value instead of  name of the color.

### Expected Result:
Narrator should be announced  name of the color.

### Attachment:
**Narrator Attachment:**
![A11y_AzureDataStudio_Create new server group_Narrator](https://user-images.githubusercontent.com/100189407/155977559-2b66397f-56e1-4a68-ad0c-42f7899c1919.gif)
**NVDA Attachment:**
![A11y_AzureDataStudio_Create new server group_NVDA](https://user-images.githubusercontent.com/100189407/155977718-ed94c98c-4955-47a6-91bf-499183a58bae.gif)

**Comment by alanrenmsft** [comment]
by design, it is more accurate to announce the color code, not every color has a name.
